include/glow/Graph/Graph.h

```diff
@@ -46,6 +46,13 @@ using PlaceholderList = std::list<Placeholder *>;
 using UnsignedArrayRef = llvm::ArrayRef<size_t>;
 /// Map from original Nodes to cloned Nodes.
 using NodeMap = llvm::DenseMap<Node *, Node *>;
+/// State of a function. This can be used to control optimizations which depend
+/// on the state of the Function.
+enum class FunctionState {
+  FuncCreated, /// Indicates that the function has been created but not
```

Comment:
We usually put comments before declarations, not in the same line.

Reply:
I looked at enum class QuantizationMode and followed the same convention. Is this documented somewhere? Looking at the code its hard to figure out. It's different at multiple places. :-)